includes/admin/class-wc-payments-admin.php

```diff
@@ -622,6 +622,12 @@ public function register_payments_scripts() {
 		);
 
 		WC_Payments::register_script_with_dependencies( 'WCPAY_PAYMENT_GATEWAYS_PAGE', 'dist/payment-gateways' );
+		$payment_method_definitions = rawurlencode( PaymentMethodUtils::get_payment_method_definitions_json() );
```

Comment:
I needed to add this because otherwise the `/wp-admin/admin.php?page=wc-settings&tab=checkout` page didn't work. That page needs `woopaymentsPaymentMethodDefinitions` to be present,

Reply:
> Ah, thanks. Should we be adding inline scripts here or down where the scripts are enqueued (down in enqueue_payments_scripts())?

I don't think it matters too much, but I now consistently colocated `wp_add_inline_script` right after `register_script_with_dependencies`.

> Also, what do you think about putting this inline script into it's own function (or something) since we're using it 3 times now?

Sure - since it's the same script content, I moved it to a `Woo_Payments_Payment_Method_Definitions` class with a `__toString` method.
This allows us to lazily call `PaymentMethodUtils::get_payment_method_definitions_json()` only if and when the script itself is enqueued.

I think I'll follow the same approach with other scripts, once this is merged.